.github/workflows/container-validation-backends.yml

```
@@ -110,3 +110,144 @@ jobs:
+        export ISTIO_GATEWAY=istio-system/ingress-alb 
+        export VIRTUAL_SERVICE_SUPPORTS_HTTPS=true
+        export DYNAMO_CLOUD=https://${NAMESPACE}.${DYNAMO_INGRESS_SUFFIX}
+        # export DOCKER_SERVER=$CI_REGISTRY_IMAGE
+        # export DOCKER_USERNAME=gitlab-ci-token
+        # export DOCKER_PASSWORD=${CI_JOB_TOKEN}
+
+        # Install dynamo env secrets
+        kubectl create secret generic hf-token-secret --from-literal=HF_TOKEN=$HF_TOKEN -n $KUBE_NS || true
+        cd ${DYNAMO_REPO}
+    - name: Deploy Dynamo Operator
+      run: |
```

actionlint (1.7.7)

undefined variable "secret". available variables are "env", "github", "inputs", "job", "matrix", "needs", "runner", "secrets", "steps", "strategy", "vars"

(expression)